src/EventService.Validation/File/RemoveFilesRequestValidator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.EventService.Data.Interfaces;
+using LT.DigitalOffice.EventService.Models.Dto.Requests.File;
+using LT.DigitalOffice.EventService.Validation.File.Interfaces;
+
+namespace LT.DigitalOffice.EventService.Validation.File;
+
+public class RemoveFilesRequestValidator : AbstractValidator<RemoveFilesRequest>, IRemoveFilesRequestValidator
+{
+  public RemoveFilesRequestValidator(
+    IEventFileRepository eventFileRepository)
+  {
+    CascadeMode = CascadeMode.Stop;
+
+    RuleFor(request => request.FilesIds)
+      .NotNull().WithMessage("List of files ids must not be null.")
+      .NotEmpty().WithMessage("List of files ids must not be empty.");

```

Comment:
As far as I remember NotEmpty() checks if it's null, default or empty

Reply:
done